.github/workflows/build-and-release-kanvas.yml

```diff
@@ -0,0 +1,126 @@
+name: Build and Publish (V2)
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "Version of Meshery (e.g. v0.0.0)"
+        required: false
+      revision:
+        description: "Extension Sub Version (e.g. 1)"
+        required: false
+      branch:
+        description: Branch of `meshery-extensions` repository to deploy.
+        required: true
+        default: master
+
+jobs:
+  release-kanvas:
+    name: Build and Release Kanvas
+    runs-on: "ubuntu-latest"
+    steps:
+      - name: Checkout Meshery
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 1
+          path: "meshery"
+          repository: meshery/meshery
+          ref: master
+          
+      - name: Checkout meshery-extensions 
+        uses: actions/checkout@v4
+        with:
+          token: ${{ secrets.GH_ACCESS_TOKEN }}
+          repository: layer5labs/meshery-extensions
+          path: "meshery-extensions"
+          fetch-depth: 1
+          ref: ${{ inputs.branch }}
+      
+      - name: Checkout meshery-cloud 
+        uses: actions/checkout@v4
+        with:
+          token: ${{ secrets.GH_ACCESS_TOKEN_CLOUD }}
```

Comment:
I added this token randomly, this doesn't exist currently.

We might need a GH pat for the cloud repo added as a secret.